<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextureCoordConvertClass</class>
 <widget class="QMainWindow" name="TextureCoordConvertClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TextureCoordConvert</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonBrowser">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browser...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditInputPath">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>32</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>InputPath</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConvert">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCurrentFile">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>491</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>72</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>OutputPath</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBrowserOutput">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browser...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOutputPath">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>616</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TextureCoordConvert.qrc"/>
 </resources>
 <connections/>
</ui>
